packages/in_app_purchase/example/android/app/build.gradle

```diff
@@ -6,6 +6,41 @@ if (localPropertiesFile.exists()) {
     }
 }
 
+// Load the build signing secrets from a local `keystore.properties` file.
+// TODO: Create release keys and a `keystore.properties` file. See `example/README.md` for more info.
+def keystorePropertiesFile = rootProject.file("keystore.properties")
+def keystoreProperties = new Properties()
+def configured = true
+try {
+    keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
+} catch (IOException e) {
+    configured = false
+    logger.error('Release signing information not found.')
+}
+
+def DEFAULT_APP_ID = "io.flutter.plugins.inapppurchaseexample"
```

Comment:
maybe make it "YOUR APP ID HERE" to make it more obvious it should be changed?

Reply:
ohh I see we can't do that as we use it as `applicationId` below, maybe worth adding some SCREAMING CAPS comment here saying you should change this.

Reply:
Done, also decided to just duplicate the string a bunch instead of having the constant. I think it's easier to see where to change it that way.